lib/visualstudio.js

```diff
@@ -156,6 +156,8 @@ function detect(options, callback) {
 						appc.subprocess.getRealName(info.vcvarsall, function (err, vcvarsall) {
 							if (!err) {
 								info.vcvarsall = vcvarsall;
+								// vcvarsall may contain space
+								vcvarsall = vcvarsall.replace(/\ /g, '^ ');
```

Comment:
I have never seen this before. The hat ^ somehow replaces with an escaped space or something?